packages/utilities/psammead-locales/moment/ig.js

```diff
@@ -0,0 +1,51 @@
+const moment = require('moment');
+
+moment.defineLocale('ig', {
+  months: 'Jenụwarị_Febụwarị_Maachị_Eprel_Mee_Juun_Julaị_Ọgọọst_Septemba_Ọktọba_Nọvemba_Disemba'.split(
+    '_',
+  ),
+  monthsShort: 'Jen_Feb_Maa_Epr_Mee_Juu_Jul_Ọgọ_Sep_Ọkt_Nov_Dis'.split('_'),
+  weekdays: 'Sọnde_Mọnde_Tuzde_Wenesde_Tọsde_Fraịde_Satọde'.split('_'),
+  weekdaysShort: 'Sọn_Mọn_Tuz_We_Tọs_Frai_Sat'.split('_'),
+  weekdaysMin: 'Su_Mọ_Tu_We_Tọ_Fr_Sa'.split('_'),
+  longDateFormat: {
+    LT: 'HH:mm',
+    LTS: 'HH:mm:ss',
+    L: 'DD/MM/YYYY',
+    LL: 'D MMMM YYYY',
+    LLL: 'D MMMM YYYY HH:mm',
+    LLLL: 'dddd, D MMMM YYYY HH:mm',
+  },
+  calendar: {
+    sameDay: '[Taa na] LT',
+    nextDay: '[Echi na] LT',
+    nextWeek: 'dddd [na] LT',
+    lastDay: '[Unyaahụ na] LT',
+    lastWeek: 'dddd [gara aga na] LT',
+    sameElse: 'L',
+  },
+  relativeTime: {
+    future: 'na %s',
+    past: '%s gara aga',
+    s: 'ntabịanya ole na ole',
+    ss: 'ntabịanya %d',
+    m: 'otu nkeji',
+    mm: 'nkeji %d',
+    h: 'otu elekere',
+    hh: 'elekere %d',
+    d: 'otu ụbọchị',
+    dd: 'Ụbọchị %d',
+    M: 'otu ọnwa',
+    MM: 'Ọnwa %d',
+    y: 'Otu afọ',
+    yy: 'Afọ %d',
+  },
+  dayOfMonthOrdinalParse: /(Nke )\d{1,2}/,
+  ordinal: function ordinal(number) {
+    return 'Nke '.concat(number);
```

Comment:
Are we sure about this? The full examples given in the translations document don't feature this word (eg. "4 Maach 2019 14:25")

Reply:
Format `LLLL` becomes `Sọnde, 14 Febụwarị 2010 15:25` so I think it's ok? what cha think?

Reply:
the ordinal I think its like `1st March` vs `1 March`, the `1st/Nke 1` logic I think is fine, just up to us in the various date formats to use it or not like `L, LL, LLL` etc

Reply:
Ah I understand now, this makes sense. Does this mean the pidgin LL, LLL, LLLL should use Do to get ordinals then?

Reply:
Ah yes, done :)

Reply:
Ah yes, done :)